### ISSUE TITLE
How can I upload a spritesheet to create a thang with /editor/thang

### ISSUE THREAD

**Issue by m15946379540** [codecombat/codecombat#4578]
I have designed some new elements, I want to add them to the checkpoints, it seems very complicated, and these two buttons can't be used. What tools should I use to do these jobs?
![Uploading 1A6B1F8C-2237-417E-B1B7-415C76453889.png…]()

**Comment by nwinter** [comment]
You currently need to be an admin to upload new animations, since it's pretty complicated. The Thangs need to be vector in either Adobe Illustrator or Adobe Animate format. If it's in Animate, we can publish for HTML5 and upload the .js export for that. If Illustrator, then we can paste the static Thang into Animate and then publish.

Are the new Thangs in that format? We might be able to help with the upload if the quality is high.

**Comment by ousc** [comment]
@nwinter I tried to  upload the '.js'  file animation  published from Adobe Animate via the button in the editor directly, but there seems something wrong with it.
![image](https://user-images.githubusercontent.com/31619017/34856318-71e36fbc-f77f-11e7-960b-3dc2ef7ceb8d.png)

**Comment by ousc** [comment]
I use the Adobe Animate 2017, and the '.js' can 't be upload.

**Comment by nwinter** [comment]
Yes. We actually parse the .js file from an older version of Adobe Animate (when it was still called Flash). We haven't updated the parser to handle the newer format from Animate. Can you email the project file you're using to maka@codecombat.com? We'll try exporting the .js from there and see how it goes.

**Comment by ousc** [comment]
Thanks a lot!

**Comment by themaka** [comment]
As Nick said, send me the file and I'll export it from the older Flash program and re-export it for you.

**Comment by ousc** [comment]
We tried using Animate CC 2015, and found that the works can be upload successfully, is there any differences from Flash to 2015? Can we use the Animate CC 2015 create new thangs?

**Comment by pickettd** [comment]
Note that in Creative Cloud there are listings for Animate CC 2015 and Animate CC 2015.1

It looks like 2015.1 is the first version where they switched the name from Flash Professional to Animate. If you start the download for Animate CC 2015 (not 2015.1) then the name of the program changes to Flash Professional (and the html/js export and .js file worked for upload to the Thang editor).
